commerce-service/src/main/java/com/liferay/commerce/internal/starter/BaseCommerceRegionsStarter.java

```diff
@@ -33,7 +33,11 @@
 	implements CommerceRegionsStarter {
 
 	@Override
-	public void start(ServiceContext serviceContext) throws Exception {
+	public void start(long userId) throws Exception {
+		ServiceContext serviceContext = new ServiceContext();
+
+		serviceContext.setUserId(userId);
+
 		CommerceCountry commerceCountry =
 			commerceCountryLocalService.fetchCommerceCountry(
 				serviceContext.getCompanyId(), getCountryIsoCode());
```

Comment:
`serviceContext.getCompanyId()` will return `0` here as is. needs a little revamp